src/main/java/blob/Ui.java

```diff
@@ -0,0 +1,54 @@
+package blob;
+
+import javafx.scene.image.Image;
+import javafx.scene.layout.VBox;
+
+import java.util.Scanner;
+
+/**
+ * Responsible for initalising all other related objects.
+ * Responsible for hosting conversation.
+ * Responsible for certain repeated chat aesthetics used throughout the chat.
+ * Constructor requires a TaskList object as an argument.
+ */
+public class Ui {
+
+    public final String unknownCommandMsg = "ERROR! Unknown command!";
```

Comment:
The constant should be in all uppercase using underscore to separate words (aka SCREAMING_SNAKE_CASE).